run.py

```diff
@@ -23,7 +23,7 @@ def str_to_bool(s):
     parser_prepare.set_defaults(run=manager.prepare)
     parser_prepare.add_argument("--repo", required=True, type=str, help="the image repo (i.e., osirrc2019/anserini)")
     parser_prepare.add_argument("--tag", default="latest", type=str, help="the image tag (i.e., latest)")
-    parser_prepare.add_argument("--save_id", default="save", type=str, help="determiens the ID of saved image (to search from)")
+    parser_prepare.add_argument("--save_to_tag", default="save", type=str, help="used to uniquely identify (part of hash) the snapshotted image after indexing")
```

Comment:
I would just say: "used to determine the tag of the snapshotted image after indexing". It's unique only relative to the repo name (and this's by design).